.gitignore

```diff
@@ -3,3 +3,4 @@
 terraform.tfstate
 terraform.tfstate.backup
 terraform.tfstate.*.backup
+vendor
```

Comment:
Just curious, what's this?

Reply:
It's how I update these things `go mod vendor` and `go mod tidy` but we don't want the `vendor` directory in the repo, so we ignore it.